.gitignore

```diff
@@ -1,4 +1,6 @@
 .cache
+.cache-main
+*.swp
```

Comment:
It's better practice to put these kinds of things in `~/.gitignore` so they don't need to be in every project.

I also recomment `*.sw[pon]`, for those cases where you have multiple edits. :)